bin/download-syntax

```diff
@@ -262,6 +262,12 @@ REPOS = (
         license_path='LICENSE.txt',
         grammars=('syntaxes/restructuredtext.tmLanguage',),
     ),
+    Repo(
+        name='redhat-developer/vscode-java',
+        version='95ffc42',
+        license_path='LICENSE',
+        grammars=('syntaxes/java.tmLanguage.json',),
+    ),
```

Comment:
I think if you run the updater it'll resort this -- I can amend that in if you want